.prettierignore

```diff
@@ -0,0 +1,4 @@
+README.md
```

Comment:
Why is this ignored as well as the `README.ru.md` file below?

Reply:
Because these files are generated automatically by using a script prepublish in package.json

Reply:
No, they are not automatically generated, also, all `prepublish` does is run `babel`:
https://github.com/metarhia/globalstorage/blob/81366e41f8d346b98cd7203b55a58348ce815d7d/package.json#L40-L41